bot/utils/randomization.py

```diff
@@ -10,11 +11,11 @@ class RandomCycle:
     The iterable is reshuffled after each full cycle.
     """
 
-    def __init__(self, iterable: t.Iterable) -> None:
+    def __init__(self, iterable: Iterable) -> None:
```

Comment:
Do we really need `-> None` here?

Reply:
None isn't even right iirc, since it returns its self?

Correct me if I'm wrong.

Reply:
`None` would be correct.

Reply:
@onerandomusername `__init__` doesn't return anything afaik (since you don't have any `return` inside).

@Den4200 yeah but is it necessary? I am not adding typehints for methods like `__str__` or `__bytes__` because return values are obvious 🙄

Reply:
Sorry I wasn't being clear; I was responding to @onerandomusername. I agree with you, it's not necessary here in this case.

Reply:
Since there are about 2x the amount of `__init__` methods without the `None` annotation than ones without, the annotation will be removed from the remaining ones.